spec/support/load_and_extend_graph_from_file.rb

```diff
@@ -1,7 +1,7 @@
 def load_and_extend_graph_from_file(filename)
   session = Tensorflow::Session.new()
   graph = Tensorflow::Graph.new()
-  graph.read(File.dirname(__FILE__)+'/../example_graphs/'+filename)
+  graph.read(File.join(File.dirname(__FILE__), '../', 'example_graphs', filename))
```

Comment:
Removing '/' from the '../' seems to work as well. Is using '../' a Ruby coding best practice? Please advice.